doc/install.text

```diff
@@ -0,0 +1,124 @@
+The following notes should allow you to install the LSST stack and the PFS DRP.
+After the installation procedure example commands are given to show you how to
+use the pipeline. The commands have been tested on an Arch Linux machine as well
+as on Mac OS X.
+
+* Install lsst binary distribution (https://pipelines.lsst.io/install/conda.html)
```

Comment:
I used a VM from https://www.digitalocean.com, but they cost real money (albeit only a few cents an hour), so I don't recommend that being a regular modus operandi.

For this purpose —  just testing whether we need Miniconda or Anaconda — I don't think we need anything special on the machine: any of the systems we have available should do the trick.

In fact, when I followed your instructions I used Miniconda and they worked fine (modulo my comments below); since Anaconda is a superset of that, I think we can conclude that either would be ok.